src/server/handle-api.js

```diff
@@ -15,26 +15,35 @@ export default ({ server, config }) => {
     path: '/api/v1/{query*}',
     handler: (req, reply) => {
 
-      const base = `${config.siteUrl}/wp-json/wp/v2`
+      const base = `${normalizePath(config.siteUrl)}/wp-json/wp/v2`
```

Comment:
Actually reading through the docs makes me think this is about relative file/folder paths? Rather than URLs